docs/uax-with-aj1/Request-to-revise-UAX-50-for-the-harmonization-with-Adobe-Japan1.md

```diff
@@ -1,4 +1,11 @@
+# Request to revise UAX 50 for the harmonization with Adobe Japan1
+
+[W3C Japanese Layout Task Force](https://www.w3.org/groups/tf/i18n-jlreq) is a group under [W3C Internationalization Interest Group](https://www.w3.org/groups/ig/i18n) and is working to share information about gaps and requirements for support of Japanese on the Web and in eBooks. We discussed about gaps between UAX 50 and implementation of vertical glyphs in Japanese fonts. This proposal was written by the group, and is endorsed by [W3C Internationalization Working Group](https://www.w3.org/groups/wg/i18n-core) and [CITPC (Council for the Promotion of Character Information Technology)](https://moji.or.jp/).

```

Comment:
Ok, here's my proposal for revision.  I'll use bolding to make it easier to spot changes, where i can.

**The** [W3C Japanese Layout Task Force](https://www.w3.org/groups/tf/i18n-jlreq) is a group under **the** [W3C Internationalization Interest Group](https://www.w3.org/groups/ig/i18n) **that** is working to share information about gaps and requirements for **the** support of Japanese on the Web and in eBooks. **The group discussed** gaps between UAX 50 and implementation of vertical glyphs in Japanese fonts, **and the result was the following** proposal. **It** is endorsed by **the** [W3C Internationalization Working Group](https://www.w3.org/groups/wg/i18n-core) and [CITPC (Council for the Promotion of Character Information Technology)](https://moji.or.jp/).